config/optional/core.entity_form_display.node.article.default.yml

```diff
@@ -123,6 +120,9 @@ content:
       size: 65
       placeholder: ''
     third_party_settings:
+      autofill:
```

Comment:
Should we also provide this configuraiton change in `thunder_update_8304` update ?